README.md

```diff
@@ -25,7 +25,7 @@ The traces are compatible with OpenTelemetry format. They are further enriched t
 ### Try Monocle with your python genAI application
 - Get latest Monocle python brary 
 ```
-    pip install monocle-observability
+    pip install monocle_apptrace
```

Comment:
pip install monocle_apptrace **or** pip install monocle-apptrace, both works.
When published to pypi or testpypi, the name of the project with '_' will be shown as '-' in the site